run.sh

```diff
@@ -99,6 +99,7 @@ DOCKER_ENV_KEYS+="SKIP_FLAKY_TESTS "
 DOCKER_ENV_KEYS+="WDM_SERVICE_URL "
 DOCKER_ENV_KEYS+="WORKSPACE "
 DOCKER_ENV_KEYS+="JS_SDK_NPM_TOKEN "
+DOCKER_ENV_KEYS+="JS_SDK_NPM_PUBLISH_TOKEN "
```

Comment:
Publish token is different than the regular token?

Reply:
Yes